plugins/woocommerce-admin/client/task-lists/fills/steps/location.tsx

```diff
@@ -91,6 +93,7 @@ const StoreLocation = ( {
 			},
 		} );
 
+		setSubmitting( false );
```

Comment:
As a result of the below comment, I wonder if we should be defensive here and wrap the updateAndPersistSettingsForGroup call in a try-catch-finally and put the setSubmitting(false) in the finally block so that if it does error, it won't just be loading forever

Reply:
Good catch, @rjchow! I've moved to using try catch in https://github.com/woocommerce/woocommerce/pull/39613/commits/a585555d924f38dedf4e1c3f90e8716c61951fe3